src/utils/internal/misc.ts

```diff
@@ -109,12 +109,16 @@ export function mapToDictionary<T>(map: Map<string, T>): Dictionary<T> {
 
 /** Recursively delete all fields with the given name. */
 // tslint:disable-next-line: no-any
-export function deleteFieldRecursively(node: any, field: string) {
+export function deleteFieldRecursively(node: any, field: string, options: {replaceWithNulls: boolean} = {replaceWithNulls: false}) {
```

Comment:
Nit: it's probably better to make this an optional param with `?` instead:

```
options?: {replaceWithNulls?: boolean}
```

and then in the function definition:

```
options = options || {};
```

This makes it easier to extend in the future. If you add another option to `options` in the future, you don't want to have to require specifying `replaceWithNulls` in the options dict, it should be optional too.

Reply:
Thanks